package.js

```diff
@@ -13,32 +13,42 @@ Package.describe({
 
 Package.registerBuildPlugin({
     name: 'UniverseModulesNPMBuilder',
-    use: ['meteor', 'underscore@1.0.4', 'ecmascript@0.1.6', 'caching-compiler@1.0.0', 'universe:modules@'+UNIVERSE_MODULES_VERSION],
+    use: [
+        'meteor',
+        'underscore@1.0.4',
+        'ecmascript@0.1.6',
+        'caching-compiler@1.0.0',
+        'universe:modules@' + UNIVERSE_MODULES_VERSION
+    ],
     sources: ['builder.js'],
     npmDependencies: {
         'browserify': '12.0.1',
-        'envify': '3.4.0',
+        //'envify': '3.4.0', breaking my packages / let user specify it themselves if they want
         'strip-json-comments': '2.0.0',
         'camelcase': '2.0.1',
         'npm': '3.4.1'
     }
 });
 
-Package.onUse(function (api) {
+Package.onUse(function(api) {
     api.versionsFrom('1.2.0.2');
     api.use([
-        'universe:modules@'+UNIVERSE_MODULES_VERSION,
+        'universe:modules@' + UNIVERSE_MODULES_VERSION,
         'isobuild:compiler-plugin@1.0.0',
         'ecmascript@0.1.6',
         'promise'
     ]);
 
-    api.use([
-        'universe:utilities-react@0.5.4',
-        'react-runtime'
-    ], ['server', 'client'], {weak: true});
+    /**
+     * wh wait, why? its not used anywhere in package --> should be in users own package
```

Comment:
because this package is mostly for react component from npm and to protect against invariant error (it's when we have react from meteor package and from npm deps in bundle )